pkg/csi_driver/utils.go

```diff
@@ -78,6 +78,7 @@ var (
 	managedSidecarRegexAR    = regexp.MustCompile(managedSidecarPatternAR)
 	managedSidecarPatternGCR = `^(gke|staging-gke|master-gke)\.gcr\.io/gcs-fuse-csi-driver-sidecar-mounter:v\d+\.\d+\.\d+-gke\.\d+.*`
 	managedSidecarRegexGCR   = regexp.MustCompile(managedSidecarPatternGCR)
+	deprecatedFlagPatterns   = regexp.MustCompile(`Flag .*? has been deprecated`)
```

Comment:
Please add comment saying this regex should match the flags marked as deprecated here: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/cfg/config.go